config/src/account_provider_config.rs

```diff
@@ -25,6 +25,7 @@ pub struct AccountProviderConfig {
     pub secret_file: Option<PathBuf>,
 
     /// Read private from env variable `STARCOIN_PRIVATE_KEY`.
+    #[serde(default)]
```

Comment:
这个是不是应该设置成 Optoin<bool>, 然后加上 
```
#[serde(skip_serializing_if = "Option::is_none")]
```
这样默认就不会变更配置文件。

Reply:
This is what #[serde(skip_serializing_if)] does
Call a function to determine whether to skip serializing this field. The given function must be callable as fn(&T) -> bool, although it may be generic over T. For example skip_serializing_if = "Option::is_none" would skip an Option that is None.

Change the type to Option is a choice, but seem like there is no need to change a boolean value to Option, right?

Reply:
有一些区别。bool 会默认序列化成 false 写到配置中。但使用的时候需要指定 `--from-env true`, 我们有一部分 bool 的 CLI 选项用的是 Option。不过 from-env 这个在配置文件中应该用不到，默认序列化成 false 也没问题。